application/controllers/InstallerController.php

```diff
@@ -972,7 +977,12 @@ private function writeConfigFile()
             . "\t" . "// on your webspace." . "\n"
             . "\t" . "// LimeSurvey developers: Set this to 2 to additionally display STRICT PHP error messages and get full access to standard templates" . "\n"
             . "\t\t" . "'debug'=>0," . "\n"
-            . "\t\t" . "'debugsql'=>0, // Set this to 1 to enanble sql logging, only active when debug = 2" . "\n";
+            . "\t\t" . "'debugsql'=>0, // Set this to 1 to enanble sql logging, only active when debug = 2" . "\n"
+            . "\n"
+            . "\t\t" . "// If URLs generated while running on CLI are wrong, you need to uncomment the following line and set your" . "\n"
+            . "\t\t" . "// public URL (the URL facing survey participants). You will also need to set the request->baseUrl in the section above." . "\n"
```

Comment:
But ? Set only publicurl work good before in CLI too ?
No ?

Reply:
What do you mean by "before"?

As to give some context, this is a followup on [18961](https://bugs.limesurvey.org/view.php?id=18961)

Reply:
In 3.X